CMIP6_source_id.json

```diff
@@ -2342,35 +2354,35 @@
             "model_component":{
                 "aerosol":{
                     "description":"SPRINTARS6.0",
-                    "nominal_resolution":""
+                    "nominal_resolution":"150 km"
                 },
                 "atmos":{
                     "description":"CCSR AGCM (T85; 256x128; 81 levels; top level 0.004 hPa)",
-                    "nominal_resolution":""
+                    "nominal_resolution":"150 km"
                 },
                 "atmosChem":{
                     "description":"none",
                     "nominal_resolution":"none"
                 },
                 "land":{
                     "description":"MATSIRO6.0",
-                    "nominal_resolution":""
+                    "nominal_resolution":"150 km"
                 },
                 "landIce":{
                     "description":"none",
                     "nominal_resolution":"none"
                 },
                 "ocean":{
                     "description":"COCO4.9 (360x256x63; tripolar primarily 1deg; 63 levels; top grid cell 0-2 m)",
-                    "nominal_resolution":""
+                    "nominal_resolution":"1 degree"
```

Comment:
As the grid resolution is not 360 longitude x 180 latitude, the appropriate `nominal_resolution` is likely `100 km` not the `1 degree` as noted above. This is also true to the `COCO4.9` entry below